api/R/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: R
+nav_order: 2
+parent: API
+has_children: true
+---
+# DataCommons R API
```

Comment:
Move this to R api PR?

Reply:
Looks like its for migrating to relative paths-- there's another page added for REST. Maybe mention this as Added python docs + changed paths to relative?

Reply:
It was just a placeholder. Deleted.